django_auth_adfs-1.11.4.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
django_auth_adfs-1.11.4.dist-info/LICENSE,sha256=n3ImkhGUnnJEu1sT51QKc_N22WdixSRcrfg9gj6nr3c,1296
django_auth_adfs-1.11.4.dist-info/METADATA,sha256=Y-xXMi4yddLMe3EWv6Ss8-jz3o2mM3Ra4_xNPtFkMnw,6305
django_auth_adfs-1.11.4.dist-info/RECORD,,
django_auth_adfs-1.11.4.dist-info/REQUESTED,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
django_auth_adfs-1.11.4.dist-info/WHEEL,sha256=vVCvjcmxuUltf8cYhJ0sJMRDLr1XsPuxEId8YDzbyCY,88
django_auth_adfs/__init__.py,sha256=euevnAm-cPW84ZlsJMIDQohV5CMe2SxqHrr9A6Rol7s,137
django_auth_adfs/__pycache__/__init__.cpython-311.pyc,,
django_auth_adfs/__pycache__/backend.cpython-311.pyc,,
django_auth_adfs/__pycache__/config.cpython-311.pyc,,
django_auth_adfs/__pycache__/drf-urls.cpython-311.pyc,,
django_auth_adfs/__pycache__/drf_urls.cpython-311.pyc,,
django_auth_adfs/__pycache__/exceptions.cpython-311.pyc,,
django_auth_adfs/__pycache__/middleware.cpython-311.pyc,,
django_auth_adfs/__pycache__/rest_framework.cpython-311.pyc,,
django_auth_adfs/__pycache__/signals.cpython-311.pyc,,
django_auth_adfs/__pycache__/urls.cpython-311.pyc,,
django_auth_adfs/__pycache__/views.cpython-311.pyc,,
django_auth_adfs/backend.py,sha256=UxKnlMrQ3IMOnIiBd3itYtFOer-QxLTSHXXt5deew-s,18055
django_auth_adfs/config.py,sha256=nC3H2iJVjgzmsZrqoWf83FEM9aSdaMbkXsG7lXtINrk,15808
django_auth_adfs/drf-urls.py,sha256=bBnN1WrgbHoiAs5hTGe-ovmVwwJ4D2xXpYtZVfE8l5U,221
django_auth_adfs/drf_urls.py,sha256=NcRbLVckCNWOXyTyTofwR7nDz7B_zNOwHAljFAI8KEw,355
django_auth_adfs/exceptions.py,sha256=yxwKHtDSMnskgBdIQoCWerYVr8sGtIn26GWVVdhA2YU,110
django_auth_adfs/middleware.py,sha256=st8Hx5pHWNcYUNWrCQht3QJqjw6RLQhWyxneTWal0Nw,2249
django_auth_adfs/rest_framework.py,sha256=spWIXpb8UtVDvZ0_W_sJB-5u0hwvLNUCqbAhHf9N0W4,1656
django_auth_adfs/signals.py,sha256=HGD79-dZNEwqVuINoqe6jB_8K1RfuK5Q288RGamTtEU,137
django_auth_adfs/templates/django_auth_adfs/login_failed.html,sha256=1NQ5aCjo7iTLSxy2IXJZbzBp5MaxpfsfQwQ_PDFvgr4,652
django_auth_adfs/urls.py,sha256=4Cec5XbxrAqPHc_N3yMTC_m9UKZXsZktf3rO2niF4Bw,533
django_auth_adfs/views.py,sha256=kwMqcsDAgcuQRBgxpRVnFZY79ZkVo28LB3KOIbayI3E,4100
